lib/smart_proxy_openscap/storage_fs.rb

```diff
@@ -26,6 +26,13 @@ def get_arf_html(digest)
       arf_object.html.force_encoding('UTF-8')
     end
 
+    def delete_arf_file
+      path = "#{@path_to_dir}/#{@namespace}/#{@cname}/#{@id}"
```

Comment:
@shlomizadok: I originally had `@path`, but that leaves us with a bunch of empty `@id/@date` folders within `@cname` directory so I decided to delete them as well since they will not be needed anymore. Would you like me to change it back to `@path`?